### PR TITLE
"What images can I use for thumbnails?"

### PR DESCRIPTION
**Pull request by indiegamehunt**

### CODE REVIEW

File: thumbnail-image/index.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+- The maximum width and height is 4096x4096. 
+- Your original image should be as large as possible (up to 4096x4096). 
+- The aspect ratio must be 16x9. If it's not 16x9, you can use the handy cropping tool that we've provided. Alternatively, you can crop it with your tool of choice before uploading. 
+    - _Tip:_ This will help if you're not using our cropping tool. You can find the necessary height by taking the image's current width and multiplying it by 0.5625. Round up. 
+- The maximum file size is 3MB. If your file is larger, try compressing and/or optimizing the image. 
+- Your file must be a PNG, JPG, GIF, or animated GIF. PNGs generally look the best. 
+
+[What is a thumbnail image?](Link)
+
+[Choosing a thumbnail image](Link)
```
Comment (MaskofThoro) [inline thumbnail-image/index.md]: might need to update the above 2 links once those two are merged into 1 question.

**Comment by indiegamehunt** [comment]
Note to self: Change heading to "What are the thumbnail requirements and guidelines?"

**Comment by hworld** [comment]
can we also combine this with #25 we shouldn't have multiple questions for almost the same thing.

**Comment by indiegamehunt** [comment]
Combining with #24 and #25

**Comment by indiegamehunt** [comment]
This is ready for review again, @MaskofThoro

File: src/thumbnail-image/index.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
+# What is a thumbnail?
+
+Your game's thumbnail is probably the most important image of your game on the site. This is the image that will appear in game listings, in search results, and so on.
+
+These little images can be the difference between someone checking out your game and barely noticing it. In many cases, a thumbnail will be the first impression someone has of your game.
+
+## What are the thumbnail requirements and guidelines?
+
+- The minimum dimensions are 640x360 (recommended minimum of 1280x720 to look decent on 4K displays).
```
Comment (MaskofThoro) [inline src/thumbnail-image/index.md]: is this still correct now that we have the cropping tool?

File: src/thumbnail-image/index.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+Your game's thumbnail is probably the most important image of your game on the site. This is the image that will appear in game listings, in search results, and so on.
+
+These little images can be the difference between someone checking out your game and barely noticing it. In many cases, a thumbnail will be the first impression someone has of your game.
+
+## What are the thumbnail requirements and guidelines?
+
+- The minimum dimensions are 640x360 (recommended minimum of 1280x720 to look decent on 4K displays). 
+- The maximum width and height is 4096x4096. 
+- Your original image should be as large as possible (up to 4096x4096).
```
Comment (MaskofThoro) [inline src/thumbnail-image/index.md]: is this still correct now that we have the cropping tool?

File: src/thumbnail-image/index.md [hunk 4]
```diff
@@ -0,0 +1,31 @@
+# What is a thumbnail?
+
+Your game's thumbnail is probably the most important image of your game on the site. This is the image that will appear in game listings, in search results, and so on.
+
+These little images can be the difference between someone checking out your game and barely noticing it. In many cases, a thumbnail will be the first impression someone has of your game.
+
+## What are the thumbnail requirements and guidelines?
+
+- The minimum dimensions are 640x360 (recommended minimum of 1280x720 to look decent on 4K displays). 
+- The maximum width and height is 4096x4096.
```
Comment (MaskofThoro) [inline src/thumbnail-image/index.md]: is this still correct now that we have the cropping tool?

File: src/thumbnail-image/index.md [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+Your game's thumbnail is probably the most important image of your game on the site. This is the image that will appear in game listings, in search results, and so on.
+
+These little images can be the difference between someone checking out your game and barely noticing it. In many cases, a thumbnail will be the first impression someone has of your game.
+
+## What are the thumbnail requirements and guidelines?
+
+- The minimum dimensions are 640x360 (recommended minimum of 1280x720 to look decent on 4K displays). 
+- The maximum width and height is 4096x4096. 
+- Your original image should be as large as possible (up to 4096x4096). 
+- The aspect ratio must be 16x9. If it's not 16x9, you can use the handy cropping tool that we've provided. Alternatively, you can crop it with your tool of choice before uploading.
```
Comment (MaskofThoro) [inline src/thumbnail-image/index.md]: is this needed now that we have the cropping tool?

File: src/thumbnail-image/index.md [hunk 6]
```diff
@@ -0,0 +1,31 @@
...
+
+These little images can be the difference between someone checking out your game and barely noticing it. In many cases, a thumbnail will be the first impression someone has of your game.
+
+## What are the thumbnail requirements and guidelines?
+
+- The minimum dimensions are 640x360 (recommended minimum of 1280x720 to look decent on 4K displays). 
+- The maximum width and height is 4096x4096. 
+- Your original image should be as large as possible (up to 4096x4096). 
+- The aspect ratio must be 16x9. If it's not 16x9, you can use the handy cropping tool that we've provided. Alternatively, you can crop it with your tool of choice before uploading. 
+    - _Tip:_ This will help if you're not using our cropping tool. You can find the necessary height by taking the image's current width and multiplying it by 0.5625. Round up.
```
Comment (MaskofThoro) [inline src/thumbnail-image/index.md]: do they have the option to not use the cropping tool? is this needed?

Comment (indiegamehunt) [inline src/thumbnail-image/index.md]: They don't have to use the cropping tool if their image is the right size. Is that correct, @hworld ?

File: src/thumbnail-image/index.md (hunk 4 above)
Comment (indiegamehunt) [inline src/thumbnail-image/index.md]: Are these dimensions still correct, @hworld ?

File: src/thumbnail-image/index.md [hunk 7]
```diff
@@ -0,0 +1,32 @@
+# What is a thumbnail?
+
+Your game's thumbnail is probably the most important image of your game on the site. This is the image that will appear in game listings, in search results, and so on.
+
+These little images can be the difference between someone checking out your game and barely noticing it. In many cases, a thumbnail will be the first impression someone has of your game.
+
+## What are the thumbnail requirements and guidelines?
+
+- The minimum dimensions are 640x360 (recommended minimum of 1280x720 to look decent on 4K displays).
```
Comment (MaskofThoro) [inline src/thumbnail-image/index.md]: this is incorrect. david will send yaprak the correct info

File: src/thumbnail-image/index.md (hunk 4 above)
Comment (MaskofThoro) [inline src/thumbnail-image/index.md]: yes, they are.

File: src/thumbnail-image/index.md (hunk 3 above)
Comment (MaskofThoro) [inline src/thumbnail-image/index.md]: this is correct

File: src/thumbnail-image/index.md (hunk 6 above)
Comment (MaskofThoro) [inline src/thumbnail-image/index.md]: yes, it's correct

**Pull request merged**